crates/sargon/src/wrapped_radix_engine_toolkit/high_level/manifest_building/account_locker/account_locker_claimable_resource.rs

```diff
@@ -0,0 +1,51 @@
+use crate::prelude::*;
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, uniffi::Enum)]
+pub enum AccountLockerClaimableResource {
```

Comment:
missing doc on enum and cases

Reply:
I was wondering whether I should write doc or not because it looks like all the types in the `wrapped-radix-engine_toolkit` are missing doc. I guess the general rule is always to write doc, right?

Reply:
yes, all new types and public methods/funcs should be documented, and hopefully we can even add doc to those symbols already missing.

for SargonFOO which is our version of ScryptoFOO or RetFOO you can copy paste the doc of Scrypto | Ret over...